Patches/Moyo light in the abyss/Ammo/Ammo_Moyo.xml

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Moyo-Light in the Abyss</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<!-- ==================== Plasma ========================== -->
+			
+			
+			
+			<li Class="PatchOperationAdd">

```

Comment:
No real reason not to just roll all the ammo files into one Add operation. It'd make it a bit more compact and would save a tiny bit of load time on the extra operations.

Reply:
Completed, merged into one op.